deliver/lib/deliver/app_screenshot.rb

```diff
@@ -322,10 +322,10 @@ def self.devices
 
     def self.resolve_ipadpro_conflict_if_needed(screen_size, filename)
       is_3rd_gen = [
-        "iPad Pro (12.9-inch) (3rd generation)", # default simulator name has this
-        "iPad Pro (12.9-inch) (4th generation)", # default simulator name has this
-        "ipadPro129", # downloaded screenshots name has this,
-        "3GEN" # downloaded screenshots name from App Store Connect API has this
```

Comment:
I'm replacing 3GEN (introduced [here](https://github.com/fastlane/fastlane/commit/eadd030370cc62807c22fbb1af3663e98e293f8c#diff-190fa1675ec92fb1d3f2323d7638bb96406dd464e58278f57eb5bc5da1889808R328) with `IPAD_PRO_3GEN_129` since it's the full name. No strong preference, though.

Reply:
Nevermind, I just saw that this change was added by @joshdholtz 😁 https://github.com/fastlane/fastlane/pull/19070/commits/29598bc859bf1828831b722a67c2e5a166002bd4

I think it's safe to include the entire identifier `IPAD_PRO_3GEN_129` 🙏 but I don't mind keeping just `3GEN` either